### DISCUSSION TITLE
Invalid CloudFormation template

### DISCUSSION THREAD

**Discussion by kevincerro** [brefphp/bref#1354] [Support]
**Description:**

I'm trying to deploy my Symfony app on new eu-south-2 region but this error appears

`Error:
The CloudFormation template is invalid: Template format error: Unrecognized resource types: [AWS::ApiGatewayV2::Api, AWS::ApiGatewayV2::Stage, AWS::ApiGatewayV2::Route, AWS::ApiGatewayV2::Integration]
`

Maybe the new region doesn't support some ApiGateway resources yet?
Probably this issue is related with serverless itself and not with bref, so can be closed if useless.

**How to reproduce:**

1. Create serverless project on eu-south-2 region
2. Error appears

Environment: darwin, node 19.2.0, framework 3.25.1 (local) 3.25.1v (global), plugin 6.2.2, SDK 4.3.2
Bref layer: php-81-fpm

**Comment by mnapoli** [comment]
Indeed I'm not sure that really relates to Bref (the region sounds like the best guess). I'll move this to a discussion, but if I were you I'd look into what is supported in eu-south-2 (not even sure how to find that 😅)

**Comment by kevincerro** [comment]
Seems that API Gateway is supported in eu-south-2, [can be checked here](https://aws.amazon.com/es/about-aws/global-infrastructure/regional-product-services/)

On the APIGateway docs eu-south-2 doesn't appear yet
https://docs.aws.amazon.com/es_es/general/latest/gr/apigateway.html#apigateway_region
